### PR TITLE
Ensure that all the tests are run and cleanup dev deps

### PR DESCRIPTION
**Pull request by lgeiger**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -6,7 +6,7 @@
   "scripts": {
     "prebuild": "mkdirp lib && mkdirp dist && npm run clean",
     "test": "zuul -- test/*",
-    "test-local": "zuul --local test/*",
+    "test-local": "zuul --local -- test/*",
```
Comment (lgeiger) [inline package.json]: Without the dashes `zuul` skipped some tests and did funny things when adding new ones.

During the debugging I upgraded all the development dependencies and removed unused ones.

**Comment by rgbkrk** [comment]
Looks good to me. Feel free to switch to another testing setup if you like.

**Comment by lgeiger** [comment]
I have zero experience with different testing setups.

What setup would you suggest?

**Comment by rgbkrk** [comment]
Back in the day we were using mocha and karma. The jupyter-js-notebook has a pretty clean setup that will work across browsers: https://github.com/jupyter/jupyter-js-notebook/blob/master/package.json#L54-L59

**Pull request merged**